test/parallel/test-http-set-timeout-server.js

```diff
@@ -152,11 +152,12 @@ test(function serverResponseTimeoutWithPipeline(cb) {
 });
 
 test(function idleTimeout(cb) {
-  const server = http.createServer(common.mustCall((req, res) => {
+  // Do not wrap the callback in common.mustCall(). It might not be invoked.
+  const server = http.createServer((req, res) => {
```

Comment:
But that's what the new test case does.
Superficially this is not wrong or harmful, but might be distracting for future maintainers.
IMHO either remove, or add to the comment that the next test case makes sure this is always true.